packages/core/scripts/hardhat/verifyDeployedContracts.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+NETWORK_NAME=$1
+if [ ! "$NETWORK_NAME" ]; 
+then 
+    echo "Must specify network name"
+    exit 1
+fi
+
+# The network to verify Etherscan contracts for
+yarn hardhat etherscan-verify --license AGPL-3.0 --force-license --network $NETWORK_NAME
```

Comment:
you don't need to tell this function what contracts you are verifying? does it just try _every_ contract? how does this work exactly.

Reply:
it just tries to verify every single contract stored in the local `deployments` folder. This is a description of the `etherscan-verify` task that the `hardhat-deploy` plugin adds to hardhat:

```
This plugin adds the etherscan-verify task to Hardhat.

This task will submit the contract source and other info of all deployed contracts to allow etherscan to verify and record the sources.
```